tee-worker/litentry/core/assertion-build/src/brc20/amount_holder.rs

```diff
@@ -46,7 +46,12 @@ pub fn build(
 	})?;
 
 	if response.is_empty() {
-		Err(Error::RequestVCFailed(Assertion::BRC20AmountHolder, ErrorDetail::NoEligibleIdentity))
+		Err(Error::RequestVCFailed(
+			Assertion::BRC20AmountHolder,
+			ErrorDetail::StfError(ErrorString::truncate_from(
+				"BRC20 token not found for linked identity".as_bytes().to_vec(),
+			)),
```

Comment:
same here. 
Do I understand it right: even there is no eligible identity, an empty/false VC will still be created, right? Here will just return error message. No empty/dummy VC.

Reply:
Fixed in this commit now: https://github.com/litentry/litentry-parachain/pull/2739/commits/9bf130b8b44100863a9ff89d3c5502bd2ceef238